disorder-core/disorder-core.cabal

```diff
@@ -17,6 +17,7 @@ library
                      , text                            >= 1.1        && < 1.3
                      , process                         == 1.2.*
                      , QuickCheck                      == 2.7.*
+                     , ieee754                         == 0.7.*
```

Comment:
@etorreborre: I'm unsure; there are no interface changes in 0.7.* to `Data.AEq` that I can see. Given 0.7.6 is in Stackage LTS 2.18 maybe we can put an upper bound there?